mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/user/CassandraSubscriptionMapper.java

```diff
@@ -58,22 +56,13 @@ public synchronized void delete(Subscription subscription) {
             .and(eq(MAILBOX, subscription.getMailbox())));
     }
 
-    @Override
-    public Subscription findMailboxSubscriptionForUser(Username user, String mailbox) {
```

Comment:
why?

Reply:
Because that method is never used anymore: StoreSubscriptionManager no longer do read-before-writes which means that getting the very stored subscription becomes an implementation detail (a JPA one btw)